docs/src/rust/user-guide/expressions/user-defined-functions.rs

```diff
@@ -9,87 +9,16 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     println!("{}", df);
     // --8<-- [end:dataframe]
 
-    // --8<-- [start:shift_map_batches]
```

Comment:
lines 4-10 aren't used, right? As in, they're currently shown

![image](https://github.com/pola-rs/polars/assets/33491632/f2a9bdbf-bf35-4927-9758-925eef3b543f)


but then all subsequent operations skip the rust example

Reply:
Yeah this is about writing custom functions from a Python programmer's perspective. If you're writing custom function in Rust you'd have to use the plugins API, map_batches() doesn't make sense.